lauraguerra1/whats-cookin
# Feature/center recipe modal and fix ingredient amounts

## Description
This pr:
- Moves the recipe modal to the center of the page and gives it a `max-width`
  - user can no longer scroll the main recipe list while viewing a recipe in the modal
- TDD for a function to remove extra decimals past 2 on ingredient amounts

Testing Notes
- Someone with a big screen, please test that the recipe modal is actually in the center for I cannot >_<
- Click on the "Pastry Cream" recipe at the bottom of the page
  - Ensure sucrose only has 2 decimal places
  - No other ingredients that have 2 or less decimal places should be affected
